timemachine/fe/single_topology.py

```diff
@@ -544,13 +559,21 @@ def concatenate(arrays, empty_shape, empty_dtype):
         # either 0 or 180, since the normalized chiral volume is still smooth wrt perturbations
         all_proper_dummy_chiral_atom_idxs_ = []
         all_proper_dummy_chiral_atom_params_ = []
+
         for (c, i, j, k), p in zip(all_dummy_chiral_atom_idxs, all_dummy_chiral_atom_params):
-            if all((c, x) in mol_c_bond_idxs_set or (x, c) in mol_c_bond_idxs_set for x in [i, j, k]):
+            missing_bonds = []
+            for x in [i, j, k]:
+                if (c, x) not in mol_c_bond_idxs_set and (x, c) not in mol_c_bond_idxs_set:
+                    missing_bonds.append((c, x))
+
+            if len(missing_bonds) == 0:
                 all_proper_dummy_chiral_atom_idxs_.append((c, i, j, k))
                 all_proper_dummy_chiral_atom_params_.append(p)
             elif verify:
+                # print("VERIFY", [(c, x) in mol_c_bond_idxs_set or (x, c) in mol_c_bond_idxs_set for x in [i, j, k]])
```

Comment:
remove debug?

Reply:
thanks - this snuck by. resolved - I modified the code so that the warnings are more informative (showing which bonds are broken in a chiral volume).